config/mk1.h

```diff
@@ -0,0 +1,75 @@
+#ifdef CONFIG_TOP

```

Comment:
Let me know when you get rid of this file, then maybe we can do the merge?

Reply:
Totally fine. So my merge was in the wrong direction between my branches resulting in this. I need to figure out how to revert a multi parent merge. 😳

Worst case I will close this Pr, cherry pick commits for a new branch and reopen the PR under a new branch.

Reply:
You can also just commit a change on top of the merge that resolves the problem.

Reply:
I will flatten the change anyway when I merge it, so it doesn't matter if the history is clean or not. :)

Reply:
Ok well in that case I will clean up my own mess later and for now just deleted the file from the branch.